packages/xo-server/src/xo-mixins/xen-servers.js

```diff
@@ -456,8 +456,9 @@ export default class {
     const xapis = this._xapis
     forEach(servers, server => {
       server.status = this._getXenServerStatus(server.id)
-      if (server.status === 'connected' && server.label === undefined) {
-        server.label = xapis[server.id].pool.name_label
```

Comment:
Do you check if `server.label` is already used in another place ?

Reply:
normally it's okay.

Reply:
[already used here](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/xo-app/settings/servers/index.js#L57)

Reply:
Yes but we need it. 
Before, `server.label` was set to match the pool name. Now, we have the real server label, otherwise, we display a placeholder that we can modify, like on my screenshot.